sdk/cosmos/test-resources.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string"
+    },
+    "enableMultipleWriteLocations" : {
+      "type": "bool",
+      "metadata": {
+        "description": "Flag to enable or disable multiple write locations on CosmosDB Account"
+      }
+    },
+    "defaultConsistencyLevel": {
+      "type": "string",
+      "metadata": {
+        "description": "Default Cosmosdb Account level consistency"
+      }
+    }
+  },
+  "variables": {
+    "apiVersion": "2020-04-01",
+    "accountName": "[toLower(parameters('baseName'))]",
+    "resourceId": "[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('accountName'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "apiVersion": "[variables('apiVersion')]",
+      "name": "[variables('accountName')]",
+      "location": "[resourceGroup().location]",
+      "kind": "GlobalDocumentDB",
+      "properties": {
+        "publicNetworkAccess": "Enabled",
+        "enableAutomaticFailover": false,
+        "enableMultipleWriteLocations": "[parameters('enableMultipleWriteLocations')]",
+        "isVirtualNetworkFilterEnabled": false,
+        "virtualNetworkRules": [],
+        "disableKeyBasedMetadataWriteAccess": false,
+        "enableFreeTier": false,
+        "enableAnalyticalStorage": false,
+        "databaseAccountOfferType": "Standard",
+        "consistencyPolicy": {
+          "defaultConsistencyLevel": "[parameters('defaultConsistencyLevel')]",
+          "maxIntervalInSeconds": 5,
+          "maxStalenessPrefix": 100
+        },
+        "locations": [
+          {
+            "locationName": "East US 2",
+            "provisioningState": "Succeeded",
+            "failoverPriority": 0,
+            "isZoneRedundant": false
```

Comment:
we should think about the multi-region accounts CI, how can we achieve that. can we have multiple arm template

Reply:
Good point Mo, @danieljurek - is there a way we can support multiple test-resources.json files - one for all single-region accounts, and other one for all multi-region accounts ?